logistics/scripts/modules/utility/xml-validation.psm1

```diff
@@ -77,12 +77,15 @@ function Invoke-XmlValidation {
     [System.Xml.Schema.XmlSchemaValidationFlags]::ReportValidationWarnings
     $xmlReaderSettings.Schemas.XmlResolver = New-Object System.Xml.XmlUrlResolver
 
+    $validationErrorCount = 0
+
     $validationHandler = [System.Xml.Schema.ValidationEventHandler] {
         if ($_.Severity -eq [System.Xml.Schema.XmlSeverityType]::Warning) {
             Write-Warning "$($_.Message)"
         }
         elseif ($_.Severity -eq [System.Xml.Schema.XmlSeverityType]::Error) {
             Write-Error "$($_.Message)"
+            $validationErrorCount++
```

Comment:
This variable only keeps value under the validation handler scope; because of this, variable from line 80 is always 0 and the exception is never thrown, failing to stop the execution.

To get this working, set the scope to script ($script:validationErrorCount) on lines 80, 88, 122 and 123

Reply:
I pushed an update with that fix